en/orm/table-objects.rst

```diff
@@ -140,6 +141,39 @@ Event List
 * ``Model.afterDelete``
 * ``Model.afterDeleteCommit``
 
+initialize
+----------
+
+.. php:method:: initialize(Event $event, ArrayObject $data, ArrayObject $options)
+
+The ``Model.initialize`` event is fired after the constructor and initialize methods are called. The table classes do not listen to this event by default, instead use an initialize hook method.
+
+Eg : Create a listener class which implements `EventListenerInterface`
+
+    use Cake\Event\EventListenerInterface;
+    class ModelInitialzieListener implements EventListenerInterface 
+    {
+        public function implementedEvents() 
+        {
+            return array(
+                'Model.initialize' => 'initializeEvent',
+            );
+        }
+        public function initializeEvent($event) 
+        {
+            $table = $event->subject();
+            // do something here
+        }
+    }
+    
+and attach the listerner to the `EventManager` as below.
```

Comment:
Preformatted blocks need to be preceded with `::` So `as below::`. I can fix that up tonight post merge.